tgui/packages/tgui/interfaces/IntegratedCircuit/VariableMenu.js

```diff
@@ -99,7 +99,7 @@ export class VariableMenu extends Component {
                             Drag me onto the circuit's grid
                             to make a setter for this variable`
                             }
-                            icon="hammer"
+                            icon="pen"
```

Comment:
maybe try download and upload signs or something?

Reply:
pen is reasonable enough imo